bootcamp_module08/core/student_11.py

```diff
@@ -23,8 +24,8 @@ def count_substring(string, substring):
     for i in range(n_subsequences):
         left_bound = i
         right_bound = i + substring_length
-        candidate_substring = string[left_bound:right_bound]
-        if candidate_substring == substring:
+        candidate_substring = string[left_bound:right_bound].lower()
+        if candidate_substring == substring.lower():
```

Comment:
You could re-arrange this so `.lower()` is called once per `string` and `substring` (outside of the `for` loop).